tests/integration/models/test_comment_forest.py

```diff
@@ -126,3 +130,18 @@ def test_replace__on_direct_comment(self):
             )
             comment.replies.replace_more()
             assert all(isinstance(x, Comment) for x in comment.replies.list())
+
+    @mock.patch("time.sleep", return_value=None)
+    def test_comment_forest_refresh_error(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+            "TestCommentForest.test_DuplicateReplaceException",
+            match_requests_on=["uri", "method", "body"],
+        ):
+            submission = self.reddit.submission(id="6aeian")
```

Comment:
Now that we have simpler STRs, can we use that instead so we have fewer requests in the cassette?

Reply:
I can change that.